backend/src/main/java/com/staccato/category/service/CategoryService.java

```diff
@@ -83,11 +79,13 @@ private List<Category> filterAndSort(List<Category> categories, List<CategoryFil
         return sort.apply(categories);
     }
 
-    public CategoryDetailResponseV2 readCategoryById(long categoryId, Member member) {
-        Category category = getCategoryById(categoryId);
+    public CategoryDetailResponseV3 readCategoryById(long categoryId, Member member) {
+        Category category = categoryRepository.findWithCategoryMembersById(categoryId)
+                .orElseThrow(() -> new StaccatoException("요청하신 카테고리를 찾을 수 없어요."));
+        ;
```

Comment:
한 곳에서만 쓰이니까 굳이 메서드화 안해도 괜찮네요!
아래쪽에 세미콜론 하나만 없애면 될 것 같습니다.

```suggestion
    public CategoryDetailResponseV3 readCategoryById(long categoryId, Member member) {
        Category category = categoryRepository.findWithCategoryMembersById(categoryId)
                .orElseThrow(() -> new StaccatoException("요청하신 카테고리를 찾을 수 없어요."));
```